DigitalLearningSolutions.Data.Tests/Helpers/TutorialContentTestHelper.cs

```diff
@@ -65,5 +95,14 @@ public void UpdateTutorialStatus(int tutorialId, int customisationId, int status
                 new { tutorialId, customisationId, status }
             );
         }

```

Comment:
Empty line between functions please. And might be worth moving this up to the other diagnostic functions? 🙂

Reply:
Yep sorry! Rearranged in 60d7021 🙂